### ISSUE TITLE
Camera bug that crashes game/client

### ISSUE THREAD

**Issue by biznessmannnn** [tgstation/tgstation#33377]
ROUND ID: 79818 
I was the head NukeOp, and when I went into the camera console for a custom placement, it froze the game for roughly 2 minutes before the client went unresponsive.

**Comment by WJohn** [comment]
There's been multiple reports on something like this, advanced cam consoles seem to be causing this a lot but it's not consistent or reproducible.

**Comment by KomradeSpectre** [comment]
Happens occassionally on/around blob overmind entities, AI Eye entities, Slime Management Console Cursor entities, as well. The screen freezes just as reported.

**Comment by praisenarsie** [comment]
i bet you it's more consistent on 512 rather than 511

**Comment by ShiggyDiggyDo** [comment]
Can confirm, happens to me too sometimes when I use advanced camera consoles like the clockcult or abductor one.

**Comment by Sosowski** [comment]
It does appear to be tied to Byond 512. My client crashed 3 times in about 5 minutes while playing AI, and after I downgraded back to 511 midround, I played a good 40 minutes without getting another crash.

**Comment by WJohn** [comment]
I think I'm gonna mark this as critical because xenobio, AI, abductor, and clock cult are largely unplayable due to this. It really needs fixing badly.

**Comment by AnturK** [comment]
Problem is this is very hard to reproduce in clean environment and without that chances of getting a fix are low.

**Comment by MrMaclang** [comment]
Happens if you're using byond 512, also saome thing happens when you are an ai

**Comment by Buggy123** [comment]
Can confirm, happens constantly as AI, not actually sure how to check version but I think i have 512.

**Comment by MrStonedOne** [comment]
I'm not going to take this report seriously until people start mentioning server.

**Comment by Jalleo** [comment]
@Buggy123 please post here with your byond version and when it next happens with the ROUND ID and SERVER NAME. And please inform if it does happen WITH 512. WE NEED DATA.

**Comment by Jalleo** [comment]
Server: Terry
Round ID: 82132
Byond Client Version 512.1403
Using a nagivation console for placing the pirate ship (whenever I overlaid the landing area with walls it crashed). It was crashing me after about 20 seconds by a freezing of the client possibly by infinite loop

**Comment by deranging** [comment]
Byond 512.1403, Sybil, Bagil, Citadel, and Hippie.

basically tg and it's close derivatives

**Comment by MrStonedOne** [comment]
1404 is out

**Comment by KomradeSpectre** [comment]
Crashes persist in 1404.

**Comment by MrStonedOne** [comment]
1404 fixed the crash we were able to reproduce, using the shuttle placer thingy. I was able to confirm this.

So what we need to do is reset, as too much info here applies to an old bug.

bugs relating to hangs on clients that involved mobs that can see all turfs have been fixed 3 times since 512 first came out.

What I need is more examples of what is going on when the hangs happen.

Close this issue report, I'll open a new one, What I need is for everybody to list times you have crashed on 1404 only. state the general area you were looking at when it happened for each time, state what kind of mob you were in, state what view mode changes you had active (mason vison, darkness toggling, huds or invisability changes, etc.)

If the turfs you had seen had changed since their round state, state any you remember, like being changed, being filled with a gas or gas overlay, being on fire, etc.
